### DISCUSSION TITLE
ArdunioOTA server?

### DISCUSSION THREAD

**Discussion by KrysFred** [JAndrassy/ArduinoOTA#101] [General]
Hey I hope all is well! I had question regarding this library. Is there a server that the ArdunioOTA.Begin is connecting to store and retrieve the information from? Or are we connecting directly to the Arduino and performing peer sharing to the device?

**Comment by JAndrassy** [comment]
for updated bin transport over network the library runs a server compatible with arduinoOTA upload tool used by Arduino IDE.
you can use the library with other binary transport by using InternalStorage object directly as shown in Advanced section of examples.
https://github.com/jandrassy/ArduinoOTA#ota-update-as-download
